lib/preprocess.js

```diff
@@ -5,20 +5,21 @@ import { CarReader } from '@ipld/car'
 import { validateBlock } from '@web3-storage/car-block-validator'
 import { recursive as exporter } from 'ipfs-unixfs-exporter'
 import createDebug from 'debug'
+import { RoundData } from './round.js'
```

Comment:
did you forget to push `round.js`? 🤔  I'm not seeing it in this PR

Reply:
Fixed, PTAL again.